styles/alpha.json

```diff
@@ -0,0 +1,228 @@
+{
+	"version": 2,
+	"title": "Alpha v3",
+	"settings": {
+		"appearanceTools": true,
+		"color": {
+			"palette": [
+				{
+					"slug": "primary-brand",
+					"color": "var(--bcds-surface-primary-default)",
```

Comment:
nice that we can just use this variable

Reply:
Agreed. For once WordPress didn't fight us on something that should be (and is) trivial ;)